packages/terra-horizontal-divider/src/HorizontalDivider.scss

```diff
@@ -0,0 +1,12 @@
+@import './variables';
+@import './mixins';
+
+:local {
+  .divider {
+    border-color: var(--terra-horizontal-divider-border-color, #b2b5b6);
+    border-style: solid;
+    border-width: var(--terra-horizontal-divider-border-width, 1px);
```

Comment:
This should probably be `border-top-width`. This is adding 1 px border to all sides.

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/5fb27563217e211a5ec708f6dc65da66fb369f31#diff-23c60d19275a3c865f09b7cdc50dbcd8L4